packages/headless/doc-parser/doc-parser.ts

```diff
@@ -257,13 +257,13 @@ const controllerConfiguration: ControllerConfiguration[] = [
     },
   },
   {
-    initializer: 'buildSearchParameterManager',
+    initializer: 'buildUrlManager',
     samplePaths: {
       react_class: [
-        'packages/samples/headless-react/src/components/search-parameter-manager/search-parameter-manager.ts',
+        'packages/samples/headless-react/src/components/url-manager/url-manager.ts',
       ],
       react_fn: [
-        'packages/samples/headless-react/src/components/search-parameter-manager/search-parameter-manager.ts',
+        'packages/samples/headless-react/src/components/url-manager/url-manager.ts',
```

Comment:
Why not add a separate configuration from the UrlManager instead of overwriting the SearchParameterManager config?

I see undocumenting a controller similar to a breaking change. I think each provides useful functionality, and they can live side-by-side,

Reply:
I thought this file was only for samples, but after seeing your other comment, I'll put it back 👍